packages/app-aco/src/contexts/acoList.tsx

```diff
@@ -208,7 +209,11 @@ export const AcoListProvider: React.VFC<AcoListProviderProps> = ({ children, ...
      */
     useEffect(() => {
         setFolders(prev => {
-            return sortTableItems(prev, state.listSort);
+            // We might receive a different field name as `title`, here we set it back to `title` for folders.
+            const titleField = props?.titleFieldId || "title";
```

Comment:
use the `id` field, as you did in https://github.com/webiny/webiny-js/pull/3554/files#diff-c38fb80874bd10f632b5189f7a787951b9f9bb5037802c93e0247a4d6b542a34R78

Reply:
👍 thanks Bruno, nice spot!